.github/tests/java-function-http+maven/assertions.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euxo pipefail
+pushd $1
+
+# Check that pom.xml exists
+test -f pom.xml
+
+# Check that build.gradle does not exist
+test ! -f build.gradle
+
+# Check that TAP files exists
+test -f config/workload.yaml
```

Comment:
Is there any combination of options where they would NOT exist? Should we focus on contents instead (rather than simply existence)?

Reply:
If `includeTap` was `false` they wouldn't exist.